src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -81,8 +81,14 @@ def __call__(self, rcvd_messages: List[Tuple[int, Message]]):
         # compute pos/neg stance on other bots using Tony's stance vector
         self.stance.get_stance()
 
+        # get dipnet order
+        orders = yield self.brain.get_orders(self.game, self.power_name)
+
         # extract only the proposed orders from the messages the bot has just received
         prp_orders = self.get_proposals(rcvd_messages)
+        
+        #include base order to prp_orders
+        prp_orders[bot.power_name] = orders
```

Comment:
Can you explain this? I am confused as to why the bots own intended orders are being included in the proposed orders.

Reply:
oh wait. I might get confused with RealPolitik. So, will this bot consider its own order if the state value of its base order (dipnet order) > any proposal orders? 

If so, I believe that we should put its base order into consideration when getting the best proposer so that we don't have to 

1. call state_value again just for its base order and 
2. then compare to the best proposer

If not, I can remove this line.

Reply:
> So, will this bot consider its own order if the state value of its base order (dipnet order) > any proposal orders?
Yes

Would you mind expanding the comment `#include base order to prp_orders`. It is not immediately apparent why we would include the bots own orders in a dict of proposed orders, so more explanation could be helpful to people reading the code later.

Reply:
I  see. I will explain this more in the comment.